python/cudf/cudf/_libxx/null_mask.pyx

```diff
@@ -19,12 +20,31 @@ from cudf._libxx.includes.null_mask cimport (
 
 from cudf.core.buffer import Buffer
 
-_mask_states = {
-    "unallocated": UNALLOCATED,
-    "uninitialized": UNINITIALIZED,
-    "all_valid": ALL_VALID,
-    "all_null": ALL_NULL
-}
+
+cdef class mask_state_value:
```

Comment:
Note: We're going to have a follow up PR refactoring this out into a preprocessed template `.pxi` file that gets included here instead so this pattern can be reused for each enum class.

Reply:
After some follow up offline discussion we decided to go with a C cast type approach to this. It's defined behavior for C++ as long as we don't go outside of the enum values which we guard against doing.